shared/actions/fs/index.js

```diff
@@ -462,23 +462,24 @@ function* ignoreFavoriteSaga(action: FsGen.FavoriteIgnorePayload): Saga.SagaGene
 //   Text/HTML;Charset="utf-8"
 //   text/html; charset="utf-8"
 // The last part is optional, so if `;` is missing, it'd be just the mimetype.
-const extractMimeTypeFromContentType = (contentType: string): string => {
+const extractMimeTypeFromContentType = (contentType, disposition: string): Types.Mime => {
```

Comment:
Would you mind either renaming `extractMimeTypeFromContentType` into `extractMimeFromContentType`, or renaming `Types.Mime` into `Types.MimeType` so they are consistent?

Reply:
Done